build/codegen/codegen.sh

```diff
@@ -14,29 +14,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+##GROUP_VERSIONS="ceph.rook.io:v1"
+
+scriptdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+basedir="$(cd "${scriptdir}/../.."  && pwd)"
+apisdir="${basedir}"/pkg/apis
+clientdir="${basedir}"/pkg/client
+
+boilerplate="${CODE_GENERATOR}/examples/hack/"boilerplate.go.txt
```

Comment:
```suggestion
boilerplate="${basedir}"/build/codegen/boilerplate.go.txt
```

Reply:
@BlaineEXE wrote:
> We should use Rook's boilerplate file, not the sample

Right. I changed it, but our file is called `header.txt`.
And I could have used `${scriptdir}"`but used the same pattern `"${basedir}"/build/codegen` as you suggested.